prog/dftb+/prg_dftb/parser.F90

```diff
@@ -3288,4 +3298,59 @@ subroutine readCustomisedHubbards(node, geo, orb, tShellResolvedScc, hubbU)
 
   end subroutine readCustomisedHubbards
 
+
+  !> Reads the parallel block.
+  subroutine readParallel(root, parallelOpts)
+
+    !> Root node eventually containing the current block
+    type(fnode), pointer, intent(in) :: root
+
+    !> Parallel settings
+    type(TParallelOpts), allocatable, intent(out) :: parallelOpts
+
+    type(fnode), pointer :: node
+
+    call getChild(root, "Parallel", child=node, requested=.false.)
+    if (withMpi .and. .not. associated(node)) then
+      call setChild(root, "Parallel", node)
+    end if
+    if (associated(node)) then
+      if (.not. withMpi) then
+        call detailedWarning(node, "Settings will be read but ignored (compiled without MPI&
```

Comment:
Why a warning instead of an error?

Reply:
This way, you can use the same input for both, serial and MPI binary. Handy for the autotest, for example.